docs/traceability/index.md

```diff
@@ -751,7 +764,96 @@ func (p *EventProcessor) Process(events []publisher.Event) []publisher.Event {
 
 ```
 
+### Traceability redaction
```

Comment:
@cbordier - please review the documentation here, it will be similar to what I would put on docs.axway.com

Reply:
@jcollins-axway this part looks strange: no formatting.
When setting up the environment variables the following is the syntax that must be used. These settings are the same as in the YAML example

TRACEABILITY_REDACTION_PATH_SHOW=[{keyMatch:"^path"}] TRACEABILITY_REDACTION_QUERYARGUMENT_SHOW=[{keyMatch:"^id$"}] TRACEABILITY_REDACTION_QUERYARGUMENT_SANITIZE=[{keyMatch:"^id$",valueMatch:"^.{0,5}"}] TRACEABILITY_REDACTION_REQUESTHEADER_SHOW=[{keyMatch:"header"}] TRACEABILITY_REDACTION_REQUESTHEADER_SANITIZE=[{keyMatch:"header",valueMatch:"."}] TRACEABILITY_REDACTION_RESPONSEHEADER_SHOW=[{keyMatch:".*"}] TRACEABILITY_REDACTION_RESPONSEHEADER_SANITIZE=[{keyMatch:"^response",valueMatch:"password"}]

Reply:
Made it a code block

Reply:
https://github.com/Axway/agent-sdk/blob/APIGOV-16534/docs/traceability/index.md#Traceability-redaction

Reply:
![image](https://user-images.githubusercontent.com/47123298/113889022-b34cf680-9777-11eb-9c77-9957d3fba1c8.png)